Services/Dashboard/templates/default/tpl.dashboard_list_block.html

```diff
@@ -1,25 +1,29 @@
 <!-- BEGIN container_row -->
 <!-- BEGIN container_header_row_image -->
-<h3 id="{BLOCK_HEADER_ID}" class="ilPDBlockSubHeader">{BLOCK_HEADER_CONTENT}</h3>
+<h3 id='{BLOCK_HEADER_ID}' class='ilPDBlockSubHeader'>{BLOCK_HEADER_CONTENT}</h3>
 <!-- END container_header_row_image -->
 <!-- BEGIN container_header_row -->
-<h3 id="{BLOCK_HEADER_ID}" class="ilPDBlockSubHeader">{BLOCK_HEADER_CONTENT}</h3>
+<h3 id='{BLOCK_HEADER_ID}' class='ilPDBlockSubHeader'>{BLOCK_HEADER_CONTENT}</h3>
 <!-- END container_header_row -->
 <!-- BEGIN container_standard_row -->
 <!-- BEGIN row_type_1 -->
-<div class="ilObjListRow">
+<div class='ilObjListRow'>
 <!-- END row_type_1 -->
 <!-- BEGIN row_type_2 -->
-<div class="ilObjListRow">
+<div class='ilObjListRow'>
 <!-- END row_type_2 -->
-<!-- <td class="std" valign="top" width="1">--><!-- BEGIN block_row_image --><!-- <img border="0" src="{ROW_IMG}" alt="{ROW_ALT}" title="{ROW_ALT}"/> --><!-- END block_row_image --><!-- {ROW_NBSP}</td>-->
-<div class="ilCLI">{BLOCK_ROW_CONTENT}</div>
+<!-- <td class='std' valign='top' width='1'>-->
+	<!-- BEGIN block_row_image -->
+	<!-- <img border='0' src='{ROW_IMG}' alt='{ROW_ALT}' title='{ROW_ALT}'/> -->

```

Comment:
Why is there a comment of an image? Is this block useless?

Reply:
Further more: i dont even know if that template is used anyway and i couldnt find any occurence of it.

Therefore i removed it.

Reply:
> If this is modified with JS there would be an ID or non generic class name. Additionally the HTML would be probably not commented out but made invisible with display none. To use a HTML comment as a JS template should not be done and is very unlikely that it is the case in this file. Please remove.

I agree that that should be the case and i wish i havent seen otherwise :smile: